### PR TITLE
Feature/unique approval numbers

### PR DESCRIPTION
**Pull request by willdot**
PBI: 72956

### CODE REVIEW

File: src/EA.Weee.RequestHandlers.Tests.DataAccess/AatfReturn/AatfTaskList/FetchAatfDataAccessTests.cs [hunk 1]
```diff
@@ -166,6 +166,51 @@ public async Task FetchAatfByReturnId_GivenReturnId_ReturnedListShouldOnlyContai
             }
         }
 
+        [Fact]
+        public async Task FetchAatfByApprovalNumber_GivenApprovalNumber_ReturnedShouldBeAatf()
+        {
+            using (var database = new DatabaseWrapper())
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.RequestHandlers.Tests.DataAccess/AatfReturn/AatfTaskList/FetchAatfDataAccessTests.cs]: Can we check AE and AATF in the test, Theory test :)

File: src/EA.Weee.Web.Tests.Unit/EA.Weee.Web.Tests.Unit.csproj [hunk 2]
```diff
@@ -358,6 +358,10 @@
       <Project>{75efee66-9358-4de4-9a84-e7191eec7db8}</Project>
       <Name>EA.Weee.Core</Name>
     </ProjectReference>
+    <ProjectReference Include="..\EA.Weee.Domain\EA.Weee.Domain.csproj">
+      <Project>{A987FFF6-5386-419E-AF0A-E6C74E3323CF}</Project>
+      <Name>EA.Weee.Domain</Name>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/EA.Weee.Web.Tests.Unit.csproj]: Shouldnt have this reference to domain from the web tests...

File: src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs [hunk 3]
```diff
@@ -315,29 +324,31 @@ private async Task<ActionResult> ManageFacilityDetails(FacilityViewModelBase vie
                     var aatf = await client.SendAsync(User.GetAccessToken(), new GetAatfById(viewModel.Id));
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs]: One last thing :) As we have the existingAatf variable we can remove this call to the API

Comment (willdot) [reply src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs]: Good spot (y)

**Pull request merged**